src/java/com/unifina/feed/StreamrBinaryMessageKeyProvider.java

```diff
@@ -1,15 +1,16 @@
 package com.unifina.feed;
 
+import com.streamr.client.protocol.message_layer.StreamMessage;
 import com.unifina.data.IStreamRequirement;
 import com.unifina.domain.data.Feed;
 import com.unifina.utils.Globals;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public class StreamrMessageKeyProvider extends AbstractKeyProvider<IStreamRequirement, StreamrMessage, String> {
+public class StreamrBinaryMessageKeyProvider extends AbstractKeyProvider<IStreamRequirement, StreamMessage, String> {
```

Comment:
The name `StreamrBinaryMessageKeyProvider` is not accurate anymore, but it is part of the database migration `2016-09-29-new-data-pipeline.groovy` and renaming it introduced errors. I don't know if it's worth it to do a migration for that or if it's fine to leave this name as is.

Reply:
Let's just leave it for now. We should remove the whole unnecessary configurability layer at some point, and this will be deleted or revisited at the time.